recipe/meta.yaml

```
@@ -1,41 +1,38 @@
-{% set version = "4.1.3" %}
+{% set version = "5.0.5" %}
 
 package:
   name: moto
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/m/moto/moto-{{ version }}.tar.gz
-    sha256: c8200ccaa9440c2e9daa0bd5e0bd768a719db5a2c82ea8d782f0e3fa09a3c5e2
+    sha256: 2eaca2df7758f6868df420bf0725cd0b93d98709606f1fb8b2343b5bdc822d91
 
 build:
   number: 0
   # there seem to be missing docker-py >=2.5.1 and cfn-lint >=0.4.0 for s390x
   skip: True  # [py<37 or s390x]
```

```suggestion
  skip: True  # [py<38 or s390x]
```
https://github.com/getmoto/moto/blob/5.0.5/setup.cfg#L27